### PR TITLE
Android’s AAPT tool added for Crunching PNG

### LINKED ISSUE #491: Crunch PNG Files
The aapt tool can optimize the image resources placed in res/drawable/ with lossless compression during the build process. For example, the aapt tool can convert a true-color PNG that does not require more than 256 colors to an 8-bit PNG with a color palette. Doing so results in an image of equal quality but a smaller memory footprint.

Keep in mind that the aapt has the following limitations:

The aapt tool does not shrink PNG files contained in the asset/ folder.
Image files need to use 256 or fewer colors for the aapt tool to optimize them.
The aapt tool may inflate PNG files that have already been compressed. To prevent this, you can use the cruncherEnabled flag in Gradle to disable this process for PNG files:
```
aaptOptions {
    cruncherEnabled = false
}
```
https://developer.android.com/topic/performance/reduce-apk-size.html

### PR DESCRIPTION
**Pull request by iml-v**
## Description
Fixes #491 Smaller PNGs, and Android’s AAPT tool added

## Related issues and discussion
#491 
 
 ## Screen-shots, if any
 
 ## Checklist
 
Make sure you've done all the following (You can delete the checklist before submitting)
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 - [x] Code is well documented
 - [x] Include unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [x] Code passes Travis builds in your branch
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

**Comment by iml-v** [comment]
@teolemon Please review this PR #1136

File: app/build.gradle [hunk 1]
```diff
@@ -45,6 +45,10 @@ android {
             buildConfigField 'String', 'OFWEBSITE', '"https://world.openpetfoodfacts.org/"'
         }
     }
+    //  Crunch PNG files
+    aaptOptions {
+        cruncherEnabled = false
```
Comment (policorn) [inline app/build.gradle]: Hi @iml-v, aapt documentation states [cruncherEnabled](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.AaptOptions.html#com.android.build.gradle.internal.dsl.AaptOptions:cruncherEnabled) option is deprecated in favor of BuildType.[isCrunchPngs](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.BuildType.html#com.android.build.gradle.internal.dsl.BuildType:crunchPngs)() - could you fix it please?

Comment (iml-v) [reply app/build.gradle]: @policorn thanks for suggestion , i will make changes .

**Comment by iml-v** [comment]
@policorn i have made changes now ,have a look at this PR #1136

**Comment by policorn** [comment]
Thanks @iml-v! Merged

**Pull request merged**